Server/VikingVault.Services/Exceptions/UserServiceException.cs

```diff
@@ -6,6 +6,5 @@ namespace VikingVault.Services.Exceptions
 {
     public class UserServiceException:Exception 
```

Comment:
Format the code to have a space before and after ":"

Reply:
Done.